tests/end_to_end/configs/test.ini

```diff
@@ -52,7 +52,7 @@ i-r-runs=-1
 i-r-fm-stop=simple
 i-r-fm-stop-i=50
 # main -> local search
-r-type=kway_fm_flow_km1
+r-type=kway_fm_km1
```

Comment:
This should be `kway_fm_hyperflow_cutter`

Reply:
updated